be/src/column/column.h

```diff
@@ -305,6 +305,8 @@ class Column {
 
     virtual void reset_column() { _delete_state = DEL_NOT_SATISFIED; }
 
+    virtual bool exceed_capacity_limit() const = 0;
```

Comment:
somewhere took up an other name, reach_capacity_limit, looks like same semantic, may be can unified into one name.
https://github.com/StarRocks/starrocks/blob/5fa55b8199f3b50fbb06ad686f979ec9d87da37d/be/src/storage/data_dir.cpp#L734

Reply:
Fixed